src/olympia/reviewers/templates/reviewers/themes/logs.html

```diff
@@ -36,12 +36,13 @@
             </tr>
             <tr class="comments hide">
               <td>&nbsp;</td>
-              <td colspan="4">
+              <td colspan="3">
                 {{ item.details.comment }}
                 {% if item.details.reject_reason %}
                   {{ REJECT_REASONS[item.details.reject_reason] }}
                 {% endif %}
               </td>
+              <td>&nbsp;</td>
```

Comment:
Is this a required change to make it work?

Reply:
@eviljeff  no, but it keeps the comment center-aligned. Earlier the comment started from 2nd column and ended up at last column (colspan=4 for .comment <td>). I thought that It looks more uniform ad nice this way otherwise it may look something like this : 
![image](https://user-images.githubusercontent.com/24241258/35326753-9fbbdcd8-011d-11e8-8c69-f76ab5bdc992.png)

Reply:
I think the extra space for long comments (meaning slightly less vertical space used) is a worthwhile trade-off.

Reply:
@eviljeff I have added an extra commit resolving that.